manifests/master.userdata_secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: master-user-data
+  namespace: openshift-machine-config-operator
```

Comment:
I actually think we should be creating this in the MAO ns, leaving a note for me to remember later